actions/image_partition_action.go

```diff
@@ -178,18 +178,19 @@ import (
 )
 
 type Partition struct {
-	number    int
-	Name      string
-	PartLabel string
-	PartType  string
-	PartUUID  string
-	Start     string
-	End       string
-	FS        string
-	Flags     []string
-	Features  []string
-	Fsck      bool "fsck"
-	FSUUID    string
+	number           int
```

Comment:
nitpick as we haven't formatted the code with gofmt yet; but the formatting is a spurious change; we should only change the line which is relevent

Reply:
So you'd rather have unaligned types for now ? It won't match the rest of the code. Should I add an additional formatting commit ?